charts/telepresence/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
I think this also needs to be `ambassador` namespace (probably the easiest solution) or change the args to be `traffic-manager.ambassador` so that it's always finds the traffic-manager in the right namespace, regardless of whichever namespace it's deployed to.  But I think i'd go with deploying it in the ambassador namespace just for ease.